AIDevGallery/Samples/WCRAPIs/BackgroundRemover.xaml

```diff
@@ -9,100 +9,72 @@
     xmlns:samples="using:AIDevGallery.Samples"
     xmlns:shared="using:AIDevGallery.Samples.SharedCode"
     mc:Ignorable="d">
-    <Grid ColumnSpacing="24" RowSpacing="12">
+    <Grid RowSpacing="12">
         <Grid.RowDefinitions>
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="*" />
-            <ColumnDefinition Width="*" />
-        </Grid.ColumnDefinitions>
+        <Viewbox 
```

Comment:
![image](https://github.com/user-attachments/assets/4a434612-d663-441d-9067-12c2bcbcbb79)

I think we need to fiddle around with the RowDefinitions here, I think the image should just scale so that the buttons are always visible?

Reply:
I think the first RowDefinition should be *, the others Auto